packages/happy-dom/src/nodes/html-element/HTMLElement.ts

```diff
@@ -200,6 +200,16 @@ export default class HTMLElement extends Element implements IHTMLElement {
 		return this._style;
 	}
 
+	/**
+	 * Sets style.
+	 *
+	 * @param cssText Style as text.
+	 * @see https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/style#setting_styles
+	 */
+	public set style(cssText: string | CSSStyleDeclaration) {
+		this.setAttribute('style', typeof cssText === 'string' ? <string>cssText : '');
```

Comment:
```suggestion
		this.style.cssText = cssText;
```

We have to parse the CSS string before setting it (e.g. `element.style = 'invalid: invalid'` won't work).

Reply:
Thanks, I updated